packages/wrangler/src/vectorize/upsert.ts

```diff
@@ -45,6 +46,11 @@ export const vectorizeUpsertCommand = createCommand({
 	},
 	positionalArgs: ["name"],
 	async handler(args, { config }) {
+		if (!(await isValidFile(args.file))) {
+			logger.error(`🚨 Cannot read invalid or empty file: ${args.file}.`);
```

Comment:
should this throw a `UserError` instead?
```
if (!(await isValidFile(args.file))) {
  throw new UserError("Could not read invalid or empty file: ${args.file}");
}
```

Reply:
Now throwing a UserError for this. The latest commit now updates the behavior for all Vectorize commands to throw UserErrors instead of logging errors and returning early.